scripts/azureml-assets/azureml/assets/release/group_test.py

```diff
@@ -40,42 +80,91 @@
     my_env['subscription_id'] = subscription_id
     my_env['resource_group'] = resource_group
     my_env['workspace'] = workspace
+    my_env['token'] = args.token.strip('"')
```

Comment:
If this is needed here, then the better solution is to prevent the quotes from being added as part of the token's value somewhere further upstream.

Reply:
Now using `eval token=$token` in the publishing script in ADO.